### PR TITLE
Asset list: Dynamic value specific to status

### LINKED ISSUE #889: Asset list: Dynamic value specific to status
## Background
When viewing the asset list, I want to see the asset's amount and the associated state, so I know the next action without opening the detail view.

## Approach
The `Amount` column shows the amount and action specific to the status.

### Asset list sort order
The asset list is sorted on the `Asset number` column, in descending order (i.e. the latest asset is shown first).

### Status and display in `Amount`column
* **Created** –  empty column
* **Ready** – Priced, nothing financed: `<Available financing amount>` & `available`
* **Ongoing** – One financing transaction: `<Outstanding amount>` & `outstanding`
* **Repaid** –  Fully repaid: `<Repaid amount>` & `repaid`
* **Closed** – Closed: `<Repaid amount>` & `repaid`

### Mockups
[Asset list example (Figma)](https://www.figma.com/file/9iwvzHuvbjlHXRWlPWRxxc/Centrifuge-app---Version-2-(Branch)?node-id=7239%3A74252)

### PR DESCRIPTION
**Pull request by Hornebom**
### Description
- adds small 'gap' to DataTable to ensure readability
- refactor 'Outstanding' column to 'Amount'

#889

### Approvals
- [ ] Dev
- [x] Product

### CODE REVIEW

File: centrifuge-app/src/components/DataTable.tsx [hunk 1]
```diff
@@ -126,6 +126,7 @@ export const DataTable = <T extends Record<string, any>>({
 const Row = styled(Shelf)<any>`
   ${({ rounded, as: comp }) =>
     css({
+      gap: '5px',
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/DataTable.tsx:129]: I'm not sure about this gap, it seems to have caused right-aligned columns not to be in line with the headers anymore

Comment (Hornebom) [reply centrifuge-app/src/components/DataTable.tsx]: Removed the 'gap' for now to keep this pr as small as possible.

**Review by onnovisser — APPROVED** [review]
💯

**Comment by EstherCentrifuge** [comment]
Looks good, thanks!

**Review by sophialittlejohn — APPROVED** [review]
🥳

**Pull request merged**